include/core/parameters_lagrangian.h

```diff
@@ -29,7 +29,8 @@ namespace Parameters
     {
       no_resistance,
       constant_resistance,
-      viscous_resistance
+      viscous_resistance,
+      epsd_resistance
```

Comment:
We should rename those in a future pr to none, constant, viscous, epsd
Could you open an issue so we don't forget